packages/components/src/disclosure/src/match.ts

```diff
@@ -15,9 +15,5 @@ export function match<TValue extends string | number = string, TReturnValue = un
             .join(", ")}.`
     );
 
-    if (Error.captureStackTrace) {
```

Comment:
That looks like a random change, can i have a one liner of what happened here?

Reply:
Ah I don't know why the typing didn't work anymore, doesn't exist on error

Reply:
<img width="812" alt="image" src="https://github.com/user-attachments/assets/c1d182a6-907c-4727-8fb1-b15b04d73deb">